_translator-files/po/de/FAQ.po

```diff
@@ -45,159 +47,161 @@ msgid ""
-msgstr ""
+msgstr "{:toc}"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:19
 msgid "</details>"
-msgstr ""
+msgstr "</details>"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:21
 #, no-wrap
 msgid "**For common problems and their solutions when using Jamulus, see the [Troubleshooting](/wiki/Client-Troubleshooting) page.**\n"
-msgstr ""
+msgstr "**Häufige Probleme und deren Lösungen bei der Verwendung von Jamulus findest du auf der [Fehlerbehebungsseite](Client-Troubleshooting).**\n"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:24
 msgid "### Is there a metronome, synchronization, or some other way of keeping in time?"
-msgstr ""
+msgstr "### Gibt es ein Metronom, eine Synchronisation oder eine andere Möglichkeit, den Takt zu halten?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:26
 msgid "No. Musicians on a Jamulus Server simply play in real time together as they would in person. If you want to have a time signal, then there are shared metronome solutions on the Internet you can try. But it’s probably best to just minimise latency so nobody has an overall delay more than about 30-50 ms."
-msgstr ""
+msgstr "Nein. Die Musiker auf einem Jamulus-Server spielen einfach in Echtzeit zusammen, so wie sie es auch persönlich tun würden. Wenn du ein Zeitsignal wünscht, kannst du im Internet Lösungen für gemeinsame Metronome ausprobieren. Aber es ist wahrscheinlich am besten, die Latenz zu minimieren, so dass niemand eine Gesamtverzögerung von mehr als 30-50 ms hat."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:28
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "Denke auch daran, dass sich alle Teilnehmer an die [Goldene Regel](/wiki/Client-Troubleshooting#der-sound-ist-gut-aber-es-ist-schwierig-im-takt-zu-bleiben) halten sollten, denn davon hängt auch ab, ob alle richtig im Takt spielen können."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:30
 msgid "### How do I know if I can join a Server? Are there rules?"
-msgstr ""
+msgstr "### Woher weiß ich, ob ich einem Server beitreten kann? Gibt es Regeln?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:32
 msgid "In general, if somebody lists a Server on one of the Public Directories provided by Jamulus by default, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some Servers may state their policies in the welcome message you will see in the chat window."
-msgstr ""
+msgstr "Wenn jemand einen Server in eines der öffentlichen Verzeichnisse einträgt, die Jamulus standardmäßig bereitstellt, akzeptiert er damit, dass jeder auf ihm spielen kann. Jamulus hat keinen Passwortschutz oder andere Authentifizierungsmechanismen eingebaut. Einige Server können jedoch in der Willkommensnachricht, die du im Chat-Fenster sehen kannst, Richtlinien angeben."
```

Comment:
Authentifizierungsmechanismen eingebaut remove eingebaut and rephrase the sentence to something like Jamulus unterstützt keinen Passwort... oder andere ...

Reply:
Corrected